packages/appeals-service-api/src/services/user.service.js

```diff
@@ -183,7 +188,7 @@ const setUserStatus = async (id, status) => {
 			.collection('user')
 			.findOneAndUpdate(
 				{ _id: new ObjectId(id) },
-				{ $set: { updatedAt: new Date(), status: status } },
+				{ $set: { confirmedAt: new Date(), status: status } },
```

Comment:
Would suggest confirmedAt is only set if status is confirmed, so it's not overwritten by other status changes

Something like:

```
let fieldsToSet = { status: status };

	if (status === STATUS_CONSTANTS.CONFIRMED) {
		fieldsToSet.confirmedAt = new Date();
	}
```

Reply:
Or can merge and do later, as I'm waiting on those statuses for one of my tickets

Reply:
should there be other datetimestamps on this document like "removedAt"? Or is it better to stick with "updatedAt"?

Reply:
Maybe just leave as is for now and can figure out in one of my PRs as I'm not sure
Only business requirement for these is to order by the confirmed at date